doc/tutorials/core/table_of_content_core.markdown

```diff
@@ -125,3 +125,13 @@ understanding how to manipulate the images on a pixel level.
     *Compatibility:* \>= OpenCV 2.4.3
 
     You will see how to use the OpenCV parallel_for_ to easily parallelize your code.
+
+-   @subpage tutorial_out_of_focus_deblur_filter
+
+    *Languages:* C++
+
+    *Compatibility:* \> OpenCV 2.0
+
+    *Author:* Karpushin Vladislav
+
+    You will learn how to recover an out-of-focus image by Wiener filter.
```

Comment:
Move tutorial from "core" to "imgproc". Image processing / filtering should go there.

Reply:
ok, i am dealing with it